main_test.go

```diff
@@ -540,6 +563,41 @@ v6 = 42
 v7 = 666.666
 k1 = 1
 k2 = 2
+New values
+v1 = test1
+v2 = 90
+v3 = 1111.1111
+v4 = test2
+v5 = 50
+v6 = 50
+v7 = 1111.1111
+k1 = 123
+k2 = 456
+`),
+	})
+
+	testPkgWithCFFI(t, pkg{
+		path: "_examples/vars",
+		want: []byte(`Initial values
+v1 = v1
```

Comment:
@sbinet Just for the `vars.__doc__`? or for all the Get/Set functions?
Only vars__doc__ is okay but for each of Get/Set functions might be a problem.
The problem of Get/Set functions docstring is that output is different between Cpy2 and CFFI.

````
# CFFI output
doc(vars.GetV1()):
'returns vars.V1'

# CPy2 output
doc(vars.GetV1()):
''
````